src/EditorFeatures/Core/Implementation/NavigateTo/NavigateToItemProvider.NavigateToItemDisplay.cs

```diff
@@ -58,6 +58,11 @@ public ReadOnlyCollection<DescriptionItem> DescriptionItems
             private ReadOnlyCollection<DescriptionItem> CreateDescriptionItems()
             {
                 var document = _searchResult.NavigableItem.Document;
+                if (document == null)
```

Comment:
Do we not trust the other data as well?  For example, do we verify the source span is within the bounds of hte document we're navigating to?